### PR TITLE
chore: release 2.0.0

### PR DESCRIPTION
**Pull request by wuespace-admin**
:robot: I have created a release \*beep\* \*boop\*
---
### ⚠ BREAKING CHANGES

* Setup "Docusaurus 2"-based documentation

### Features

* :wrench: Add `ebnf` as highlighted language within `docusaurus.config.js` ([e8c9851](https://www.github.com/wuespace/telestion-docs/commit/e8c985128e141c7a9e7f3961db2ebaa33343aad8)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* Add "Reviewing and Merging Pull Requests" task for telestion-client repo ([3b6f795](https://www.github.com/wuespace/telestion-docs/commit/3b6f795d46f50837dea8c66df5d0c12f001ac7e7))
* **components:** :sparkles: Add `<FileDownload />` component ([8fe4529](https://www.github.com/wuespace/telestion-docs/commit/8fe4529a5ebd6449deb09aea45155f6e1433f6e4))
* **deployment:** :rocket: Deploy to GitHub Pages ([c9299ef](https://www.github.com/wuespace/telestion-docs/commit/c9299ef5a34563f30cfaf0ce5f713e826bed80fc)), closes [#85](https://www.github.com/wuespace/telestion-docs/issues/85)
* **docs/client:** :memo: Add `nodejs.mdx` concept ([42d0549](https://www.github.com/wuespace/telestion-docs/commit/42d05497eedc6e0744c1cd763500fe84b685e1d0)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **docs/client:** :memo: Add `npm` concept ([9b07597](https://www.github.com/wuespace/telestion-docs/commit/9b075972897b10561054000ea4d2478455f87953)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **docs/client:** :memo: Add `tc-cli` concept and general client documentation category ([d7a3b72](https://www.github.com/wuespace/telestion-docs/commit/d7a3b72f1e1bc56fedccdcbce4c357d9a4ef4aa4)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **docs/client:** :memo: Add external resources to `tc-cli.mdx` ([56425d9](https://www.github.com/wuespace/telestion-docs/commit/56425d9e8ff5bfc174a2a8d5e3eeda933403dc2d))
* **docs/internal:** :memo: Add branch naming reference ([1db51d2](https://www.github.com/wuespace/telestion-docs/commit/1db51d2aae684ec589e98fa4d0c65c91f7e499f3)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **docs/internal:** :memo: Add conventional commit type reference ([14a7dbc](https://www.github.com/wuespace/telestion-docs/commit/14a7dbc8770babfc95d405bf3030b4731b7dfe7a)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **docs/internal:** :memo: Add conventional commits concept ([dcb653b](https://www.github.com/wuespace/telestion-docs/commit/dcb653b5f9cc301a7539502811fb988212616464)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **docs/internal:** :memo: Add key git principles reference ([2a03f2a](https://www.github.com/wuespace/telestion-docs/commit/2a03f2ada0050d969d9ac317b48ef5dee343c10c)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **docs/internal:** :memo: Add Maintainer's tasks reference ([81b5abd](https://www.github.com/wuespace/telestion-docs/commit/81b5abd82a008678036809fbcb7ec44f713d4529)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **docs/telestion-client-internal:** :memo: Add `lerna` concept and general `telestion-client`-internal documentation section ([e9d983e](https://www.github.com/wuespace/telestion-docs/commit/e9d983e14f71d3691f2f0fb447e49965600a46d6)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* Setup "Docusaurus 2"-based documentation ([016b1bb](https://www.github.com/wuespace/telestion-docs/commit/016b1bb82f45e017e637be970fc4dc363ffb8177)), closes [#80](https://www.github.com/wuespace/telestion-docs/issues/80)
* **templates:** :seedling: Add category folder template `_templates/category` ([0014d0c](https://www.github.com/wuespace/telestion-docs/commit/0014d0cbc3f1945d4a611ddaacd78f1583346b35))
* **templates:** :seedling: Add concept article template `_templates/concept.mdx` ([1a89f8e](https://www.github.com/wuespace/telestion-docs/commit/1a89f8e4039847985f71617cd80e473b8ac86b7c)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **templates:** :seedling: Add guide article template `_templates/guide.mdx` ([a196040](https://www.github.com/wuespace/telestion-docs/commit/a1960402d6abb08d9c7a4ecb5719768e968a4927)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **templates:** :seedling: Add reference article template `_templates/reference.mdx` ([27e301a](https://www.github.com/wuespace/telestion-docs/commit/27e301a5c84b48bad31d93c3532176df9bacaa01)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **templates:** :seedling: Add tutorial article template `_templates/tutorial.mdx` ([0f1ec17](https://www.github.com/wuespace/telestion-docs/commit/0f1ec17028b6380584becc83c504aecede82c562)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **templates:** :seedling: Improve "Meta Template" (template for templates) ([558afd1](https://www.github.com/wuespace/telestion-docs/commit/558afd1b87a190cb0f69b32cb766e22bc9ea54f6)), closes [#79](https://www.github.com/wuespace/telestion-docs/issues/79)
* **templates:** :seedling: Make meta template more useful `_templates/_meta-template.mdx` ([72ebeb7](https://www.github.com/wuespace/telestion-docs/commit/72ebeb7720691e5f871b11bdb76b254355976650))

### Bug Fixes

* :wrench: Fix "Edit this page" link base url ([cb8d562](https://www.github.com/wuespace/telestion-docs/commit/cb8d5623c16432647c483747505a259f5c313bdf))
* **docs/client:** :mag: Add SEO description to `tc-cli.mdx` ([8f09b16](https://www.github.com/wuespace/telestion-docs/commit/8f09b16151c786361b252504c2f27dd2ff1f6bcb))
* **docs/general:** :bug: Fix broken links in `intro.mdx` ([934f0ac](https://www.github.com/wuespace/telestion-docs/commit/934f0acadae8be97c916e0d29ceee6cc05ca989c))
* **docs/internal:** :pencil2: Fix typo in conventional commit types diagram ([8365829](https://www.github.com/wuespace/telestion-docs/commit/836582996bd26e9f395977f2bf203c1c8a236561))
* **stylesheet:** :fire: Remove unneeded, commented out piece of CSS ([7b37ce9](https://www.github.com/wuespace/telestion-docs/commit/7b37ce97d7cd1c020ef9fe14cf9a6d345897aad6))
---

This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

**Review by pklaschka — APPROVED** [review]

**Review by pklaschka — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,48 +1,48 @@
...
-		"development": [
-			"last 1 chrome version",
-			"last 1 firefox version",
-			"last 1 safari version"
-		]
-	},
-	"devDependencies": {
-		"prettier": "^2.3.2"
-	}
+  "name": "docusaurus-2-test",
```
Comment (fussel178) [inline package.json:2]: What's about a really nice package name :stuck_out_tongue_winking_eye:?

```suggestion
  "name": "telestion-docs",
```

**Review by fussel178 — CHANGES_REQUESTED** [review]
Beep, boop :smile:

**Review by pklaschka — APPROVED** [review]

**Review by fussel178 — APPROVED** [review]
LGTM :palm_tree:

**Comment by wuespace-admin** [comment]
:robot: Release is at https://github.com/wuespace/telestion-docs/releases/tag/v2.0.0 :sunflower:

**Pull request merged**